layouts/partials/flex/body-beforecontent.html

```diff
@@ -3,51 +3,38 @@
     {{ partial "header.html" . }}
   </div>
   {{- if not .Site.Params.disableSearch}}
-    {{ partial "search.html"}}
+  {{ partial "search.html"}}
   {{- end}}
-  <div class="burger"><a href="javascript:void(0);" style="font-size:15px;">&#9776;</a></div>
   {{- with .Site.Menus.shortcuts}}
-    <nav class="shortcuts">
-        {{- range sort . "Weight"}}
-            <li class="" role="">
-                <a href="{{.URL}}" {{if eq $.Site.Params.menushortcutsnewtab true}}target="_blank"{{end}} rel="noopener">
-                  {{safeHTML .Name}}
-                </a>
-            </li>
-        {{- end}}
-    </nav>
+  <nav class="shortcuts">
+    {{- range sort . "Weight"}}
+    <li class="" role="">
+      <a href="{{.URL}}" {{if eq $.Site.Params.menushortcutsnewtab true}} target="_blank" {{end}} rel="noopener">
+        {{safeHTML .Name}}
+      </a>
+    </li>
+    {{- end}}
+    <li class="burger"><a href="javascript:void(0);" style="font-size:1.5rem;">&#9776;</a></li>
+  </nav>
   {{- end}}
 </header>
-<article>
-  <aside>
-    <ul class="menu">
-      {{- if not .Site.Params.disableHomeIcon}}
-          <li data-nav-id="{{"/" | relLangURL}}" class="dd-item">
-          <a href="{{"/" | relLangURL}}">
-            Wiki Home <i class="fa fa-fw fa-home"></i>
-          </a>
-          </li>
-      {{- end}}
 
-      {{- partial "menu.html" . }}
+<aside>
```

Comment:
Pulls the aside out of the article, since it's being used as a top-level menu, not part of an article. The high-level structure of the document should now be much more clear.
<img width="584" alt="screen shot 2019-02-06 at 11 44 30 am" src="https://user-images.githubusercontent.com/3818920/52357700-9801c780-2a04-11e9-85d7-6a2572b3d435.png">